src/main/resources/templates/facilities.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
There was no Spring MVC template for the controller to reference. I recommend adding this, so your controller has an HTML template mapping for that given controller endpoint.